src/views/Team/components/Footer/Footer.scss

```
@@ -0,0 +1,27 @@
+@use "~stylesheets/responsiveness" as rsp;
+@use "~stylesheets/typography" as tpy;
+
+.HeroFooter {
+  min-height: 5em;
+  display: flex;
+  flex-direction: column;
+  margin-top: 2em;
+
```

```suggestion
  padding: var(--extra-extra-large-spacer);
```
For a more consistent pad with the footer from the homepage.

---

Actually we need this in order to right align the `2021 CH` text and to center the arrow.